.travis.rosinstall

```diff
@@ -1,3 +1,4 @@
 - git:
-    uri: https://github.com/ros/std_msgs.git
```

Comment:
This will be used on indigo as well.
Please add a new `.travis.rosinstall.kinetic` file for this purpose, this will test this feature as well :)